### PR TITLE
Split up input method and valid_input method

### PR DESCRIPTION
**Pull request by chrisstankus99**
Split up input method and valid_input method into cruiser_input and submarine_input. Also split up valid input to get expected result. 

Everything has been manually tested. Simplecov shows 77.89% coverage, missing tests in output and game.

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -41,44 +42,62 @@ def start
...
+    @user_board.place(@submarine, user_input)
+    puts @user_board.render(true)
+    @user_board.place(@submarine, user_input)
     #board should render here with cruiser and submarine placed on whatever
     #coordinates were passed in
     turn
   end
 
-
   def turn
```
Comment (tylertomlinson) [inline lib/game.rb:90]: I'm currently working on this turn method. no worries

File: lib/game.rb [hunk 2]
```diff
@@ -98,16 +117,20 @@ def turn
   end
 
   def sanitized_input(user_input)
-    user_input.upcase.strip.gsub(" ", "").scan(/../)
+    user_input = user_input.upcase.strip.gsub(" ", "").scan(/../)
   end
 
-  def valid_input?(user_input)
+  def valid_cruiser_input?(user_input)
```
Comment (tylertomlinson) [inline lib/game.rb:123]: Smart to break them up. Might have to look at this though if we decide to add more ships.

File: test/game_test.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+require_relative 'test_helper'
```
Comment (tylertomlinson) [inline test/game_test.rb:1]: These tests look solid. Will look at see what is not covered and work on that.

File: lib/game.rb (hunk 2 above)
Comment (chrisstankus99) [reply lib/game.rb:123]: Good call, this could definitely be done in one method. We should look at refactoring later on.

File: lib/game.rb (hunk 1 above)
Comment (chrisstankus99) [reply lib/game.rb:90]: Ya love to see it.

**Pull request merged**